### DISCUSSION TITLE
Java(TM) Platform SE binary has stopped working

### DISCUSSION THREAD

**Discussion by pbmunck** [dcm4che/dcm4che#866] [Q&A] [answered]
try to run findscu (or another program). Then I get a window popup : Java(TM) Platform SE binary has stopped working

java version 8 update 271.

do I need to configure something?

Thanks Peter

**Comment by gunterze** [comment, marked as answer]
Guess there is something wrong with your Java Installation on your OS. You may try re-install/upgrade your Java SE Installation.

**Reply by pbmunck** [reply]
thanks! I will try...

**Reply by pbmunck** [reply]
Fixed! thanks for the fast response!
